src/query/vacuum.h

```diff
@@ -137,13 +122,7 @@ struct vacuum_worker
   // page buffer private lru list
   int private_lru_index;
 
-  /* Caches postpones to avoid reading them from log after commit top operation with postpone. Otherwise, log critical
-   * section may be required which will slow the access on merged index nodes. */
-  VACUUM_CACHE_POSTPONE_STATUS postpone_cache_status;
-  char *postpone_redo_data_buffer;
-  char *postpone_redo_data_ptr;
-  VACUUM_CACHE_POSTPONE_ENTRY postpone_cached_entries[VACUUM_CACHE_POSTPONE_ENTRIES_MAX_COUNT];
-  int postpone_cached_entries_count;
+  log_postpone_cache m_log_postpone_cache;
```

Comment:
since tdes has m_log_postpone_cache, is it useful to have one inside vacuum worker?

Reply:
fixed